runtime/onert/api/nnfw/src/nnfw_api_internal.cc

```diff
@@ -1758,8 +1758,9 @@ NNFW_STATUS nnfw_session::quantize()
     auto model = loadModel(_quant_manager->exportModelPath(), "circle");
     if (model == nullptr)
       return NNFW_STATUS_ERROR;
-    // TODO: Update _model_path if necessary
     _nnpkg->replaceModel(std::move(model));
+    _state = State::MODEL_LOADED;
```

Comment:
(Q) Is there any possible case that the `_state` is changed?

Reply:
Full quantization after minmax data collection.

`nnfw_load_model_from_file(session, ...)`: `MODEL_LOADED` -> `nnfw_prepare(session)`: `MODEL_PREAPRED` -> `nnfw_set_execute_config(session, NNFW_RUN_CONFIG_DUMP_MINMAX, nullptr)` -> `nnfw_run(session)`: `FINISHED_RUN` -> `nnfw_run(session)` -> `nnfw_run(session)` -> ... -> `nnfw_set_quantize_type(session, ...)` -> `nnfw_quantize(session)` -> `nnfw_codegen(session, ...)`